package.json

```diff
@@ -4,71 +4,75 @@
     "scripts": {
         "help:help": "echo Prints a list of available scripts and a reminder about how to get more help.",
         "help": "echo Available scripts -- execute \\`npm run help:SCRIPTNAME\\` for help on SCRIPTNAME: && (npm run | grep '^\\s\\s\\S' | grep -v 'help\\|only\\|postinstall\\|prepare')",
-        "help:dev": "run-s -s help:dev:*",
-        "help:dev:1": "echo Compiles frontscope and starts a server running it.",
-        "help:dev:2": "echo Note the server will hot-reload the code from any",
-        "help:dev:3": "echo source files as they change, to ease development.",
+        "help:dev": "run-s -s help:dev:[1-3]",
+        "help:dev:1": "echo \"Compiles frontscope and starts a server running it.\"",
+        "help:dev:2": "echo \"Note the server will hot-reload the code from any\"",
+        "help:dev:3": "echo \"source files as they change, to ease development.\"",
+        "help:dev:workbench": "run-s -s help:dev:workbench:[1-3]",
+        "help:dev:workbench:1": "echo \"Compiles frontscope and starts a server running it in workbench mode,\"",
+        "help:dev:workbench:2": "echo \"with the visualizers in the 'visualizers-workbench' directory loaded.\"",
         "dev": "npm run typecheck && vite",
+        "dev:workbench": "npm run typecheck && vite --mode workbench",
         "help:doc:serve": "run-s -s help:doc:serve:*",
-        "help:doc:serve:1": "echo This is like the 'dev' script, but for the",
-        "help:doc:serve:2": "echo embedded documentation rather than the",
-        "help:doc:serve:3": "echo Numberscope interface itself.",
+        "help:doc:serve:1": "echo \"This is like the 'dev' script, but for the\"",
+        "help:doc:serve:2": "echo \"embedded documentation rather than the\"",
+        "help:doc:serve:3": "echo \"Numberscope interface itself.\"",
```

Comment:
Why quotes on all of them? None of the other help lines have quotes, and they are a little confusing because they are unnecessary on the second two lines. Maybe just use `\\'dev\\'` in that one line?

Reply:
Great, only one line changed now, thanks.